js/form.js

```diff
@@ -1,58 +1,71 @@
+  //errorClass: '', // Класс, обозначающий невалидное поле
+  //successClass: '', // Класс, обозначающий валидное поле
+  errorTextParent: 'ad-form__element', // Элемент, куда будет выводиться текст с ошибкой
+  //errorTextTag: '', // Тег, который будет обрамлять текст ошибки
+  errorTextClass: 'ad-form__error-text' // Класс для элемента с текстом ошибки
+},);
+
+const adRooms = adForm.querySelector('#room_number');
+const adCapacity = adForm.querySelector('#capacity');
+const roomsCapacityOption = {
+  '1': ['1'],
+  '2': ['2', '1'],
+  '3': ['3', '2', '1'],
+  '100': ['0']
 };
-const cardTemplate = document.querySelector('#card').content.querySelector('.popup');
-
-const getCardElement = (card) => {
-  const cardOfferCopy = cardTemplate.cloneNode(true);
-
-  const setContent = (selector, content) => {
-    const element = cardOfferCopy.querySelector(selector);
-    if (content) {
-      element.textContent = content;
-      element.src = content;
-    } else {
-      element.remove();
-    }
-  };
-
-  setContent('.popup__title',card.offer.title);
-  setContent('.popup__text--address', card.offer.address ? `${card.offer.address}`: false);
-  setContent('.popup__text--price',card.offer.price ? `${card.offer.price} ₽/ночь`: false);
-  setContent('.popup__type', card.offer.type ? OFFER_TYPES[card.offer.type] : false);
-  setContent('.popup__text--capacity', card.offer.rooms ? `${card.offer.rooms} комнаты для ${card.offer.guests} гостей` : false);
-  setContent('.popup__text--time', card.offer.checkin ? `Заезд после ${card.offer.checkin}, выезд до ${card.offer.checkout}` : false);
-  const featuresList = cardOfferCopy.querySelectorAll('.popup__feature');
-  featuresList.forEach((featureElement) => {
-    let exist = false;
-    card.offer.features.forEach((feature) => {
-      if (featureElement.classList.contains(`popup__feature--${feature}`)) {
-        exist = true;
-      }
-    });
-    if (!exist) {
-      featureElement.remove();
-    }
-  });
-  setContent('.popup__description', card.offer.description);
-  const cardPhotos = cardOfferCopy.querySelector('.popup__photos');
-  const cardPhoto = cardOfferCopy.querySelector('.popup__photo');
-  const collectionPhoto = card.offer.photos;
-  for (let i = 0; i < collectionPhoto.length; i++) {
-    const cloneCardPhoto = cardPhoto.cloneNode(true);
-    cloneCardPhoto.src = collectionPhoto[i];
-    cardPhotos.append(cloneCardPhoto);
+
+const validateRoomsCapacity = () => roomsCapacityOption[adRooms.value].includes(adCapacity.value);
+
+const getRoomsCountErrorMessage = () => {
+  if (adRooms.value === '1') {
+    return '1 комната для 1 гостя' ;
+  }
+  if (adRooms.value === '2') {
+    return '2 комнаты для 2 гостей или 1 гостя';
+  }
+  if (adRooms.value === '3') {
+    return '3 комнаты для 3 гостей, 2 гостей или 1 гостя';
+  }
+  if (adRooms.value === '100') {
+    return 'Не для гостей';
   }
-  cardPhoto.remove();
+};
```

Comment:
Это нормально, но к слову, для таких вот случаев case подходит даже лучше, потому что не надо писать break 
```suggestion
    const getRoomsCountErrorMessage = () => {
  switch (adRooms.value) {
    case '1':
      return '1 комната для 1 гостя';
    case '2':
      return '2 комнаты для 2 гостей или 1 гостя';
    case '3':
      return '3 комнаты для 3 гостей, 2 гостей или 1 гостя';
    case '100':
    default:
      return 'Не для гостей';
  }
};
```